api/proto/metrics/metrics.proto

```diff
@@ -144,6 +145,27 @@ message appMetric {
   repeated appDiskMetric disk = 6;
 }
 
+message logMetric {
+  uint64 numDeviceLogsSent = 1;
+  uint64 numDeviceBundlesSent = 2;
+  uint64 numAppLogsSent = 3;
+  uint64 numAppBundlesSent = 4;
+  uint64 num4xxResponses = 5;
+  google.protobuf.Timestamp lastDeviceLogSendTime = 6;
+  google.protobuf.Timestamp lastAppLogsSendTime = 7;
+
+  bool isLogProcessingDeferred = 8;
+  uint64 numTimesDeferred = 9;
+  google.protobuf.Timestamp lastLogDeferTime = 10;
+
+  uint64 avgBundleLogCount = 11;
+  uint64 avgBundleProtoByteCount = 12;
+  uint64 numBundleOverFlows = 13;
+
+  uint64 totalDeviceLogInput = 14;
+  uint64 totalAppLogInput = 15;
```

Comment:
How would they different from the Sent ones?

Reply:
These count the incoming logs from rsyslogd

Reply:
And what does it mean if they are different than the sent? That something got dropped?
I'm asking what the intended usage is. We need to be able to detect drops, but you have a drop counter, right?

Reply:
My bad. I had the drop/error counters in another sandbox (along with code that fills them). Sent a slightly less updated one for review. I'll add drop counters also.